hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/impl/DirMarkerTracker.java

```diff
@@ -34,20 +34,20 @@
  * Tracks directory markers which have been reported in object listings.
  * This is needed for auditing and cleanup, including during rename
  * operations.
- * <p></p>
+ *
```

Comment:
these should actually be <p> without the closing element, but retained. 

if you set your IDE to format javadocs, it will then preserve the line break